src/common/Cargo.toml

```diff
@@ -21,7 +21,7 @@ futures = { version = "0.3", default-features = false, features = ["alloc"] }
 futures-async-stream = "0.2"
 humantime = "2.1"
 itertools = "0.10"
-lru = { git = "https://github.com/risingwavelabs/lru-rs.git", rev = "e0e9ddaf8e4a51c244a03676734437e68a336b30" }
+lru = { git = "https://github.com/risingwavelabs/lru-rs.git", branch = "evict_by_timestamp" }
```

Comment:
Have the changes to `lru-rs` been approved?

Reply:
No, let's bump it later after I optimize the lru-rs